core/src/main/resources/hudson/model/StringParameterDefinition/config.jelly

```diff
@@ -35,4 +35,7 @@ THE SOFTWARE.
     <f:entry title="${%Description}" help="/help/parameter/description.html">
         <f:textarea name="parameter.description" value="${instance.description}" codemirror-mode="${app.markupFormatter.codeMirrorMode}" codemirror-config="${app.markupFormatter.codeMirrorConfig}" previewEndpoint="/markupFormatter/previewDescription" />
     </f:entry>
-</j:jelly>
\ No newline at end of file
+    <f:entry field="trim" help="/help/parameter/trim.html">
```

Comment:
This is an obsolete help inclusion approach but it's fine for me since other fields use the same. Somebody can refactor the entire class later